examples/ASP.NET Core 2/Visual Studio 2017/ASP.NET Core 2 - VS2017/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using NLog;
+using NLog.Config;
+using NLog.Web;
+
+namespace ASP.NET_Core_2___VS2017
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // NLog: setup the logger first to catch all errors
+            var logger = NLogBuilder.ConfigureNLog("NLog.config").GetCurrentClassLogger();
```

Comment:
@snakefoot updated it! I think is indeed a lot better.

@grokky1 any opinion on this? Do you think this is clear?